lib/pages/home/wallet/wallet.dart

```diff
@@ -57,12 +57,19 @@ class WalletPage extends StatelessWidget {
     );
   }
 
+  var rowCounter = 1;
+  var currencyRowColour = Colors.white;
   Widget _buildCurrencyRow(BuildContext context, Currency currency) {
+    currencyRowColour = (rowCounter % 2 == 0 ? currencyRowColour = Colors.amberAccent.shade100 : currencyRowColour = Colors.transparent);
```

Comment:
A color like light gray is probably easier on the eyes than amber. Try something like Colors.white10.
Also keep in mind that amber doesn't look right when using the dark theme. Colors.white10 will probably look more in place when using dark theme. Alternatively you can check which theme is selected and determine the color based on that.

Reply:
I've re-done this with theme-respecting colours.